app/components/slack-context-menu.js

```diff
@@ -138,7 +149,7 @@
 	}
 
 	module.exports = {
-		handleRightClick: function (window, target, x, y) {
+		handleRightClick: function (window, target, x, y, teamUrl) {
```

Comment:
I don't quite follow what you mean; `teamUrl` is already coming from a React context =/